docs/development/debugging.md

```diff
@@ -0,0 +1,49 @@
+# Electron Debugging
```

Comment:
Note: this will need to be added to `docs/README.md` to appear in the table of contents of the old website.